source/release-notes.txt

```diff
@@ -10,10 +10,37 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.42.5
+----------------
+
+*Released April 08, 2024*
+
+New Features
+
+- Updates atlas login screen flow (:issue:`COMPASS-7755`)
+- Handles collection subtab from link (:issue:`COMPASS-7731`)
+- Z-indexed stacked components (:issue:`COMPASS-7732`)
+- Removes collapsing/expanding the sidebar (:issue:`COMPASS-7812`)
```

Comment:
[Question]

Is "Removes collapsing/expanding the sidebar" a feature?

Reply:
Max put it in the list, but I updated the language to make this clearer.